test/unit/org/apache/cassandra/cql3/validation/operations/InsertTest.java

```diff
@@ -132,12 +132,13 @@ public void testInsert() throws Throwable
     private void testInsert(boolean forceFlush) throws Throwable
     {
         createTable("CREATE TABLE %s (partitionKey int," +
+                                      "pk2 int," +
```

Comment:
No strong opinion here, but what was the reasoning behind modifying the `testInsert` here to use a composite primary key? Specifically as part of this patch for constraints here.

Reply:
Mmmmm, probably was modified unintentionally during the rebase... This is weird. I'll revert back anyway.